script/github-actions/filter-files-changed.js

```diff
@@ -0,0 +1,12 @@
+const args = process.argv.slice(2);
+const FILES = args[0].slice(1, -1).split(','); // remove unnecessary characters
```

Comment:
As a matter of convention, I typically see all-caps variables used on values that aren't calculated or derived (such as literals) or "true" constants to be used widely. I would prefer to see it declared as `const files` here.